### DISCUSSION TITLE
Gemini gets cut off after half a sentence roughly 60% of the time

### DISCUSSION THREAD

**Discussion by AumCoin** [danny-avila/LibreChat#4358] [Help Wanted]
![image](https://github.com/user-attachments/assets/2189609c-7933-4158-8191-0555bc70c55e)

I get incomplete responses after about 1s, roughly 60% of the time.  Not sure what's going on.

**Comment by JeongJuhyeon** [comment]
Are you calling it through vertex? https://github.com/danny-avila/LibreChat/discussions/4283 https://github.com/danny-avila/LibreChat/issues/4440
